docs/Reference/API-Objects.md

```diff
@@ -23,7 +23,7 @@ Returned by [eth_getBlockByHash](API-Methods.md#eth_getblockbyhash) and [eth_get
 | **miner**            | Data, 20&nbsp;bytes   | Address to which mining rewards were paid.                                                                                       |
 | **difficulty**       | Quantity, Integer     | Difficulty for this block.                                                                                                       |
 | **totalDifficulty**  | Quantity, Integer     | Total difficulty of the chain until this block.                                                                                  |
-| **extraData**        | Data                  | Extra data field of this block.                                                                                                  |
+| **extraData**        | Data                  | Extra data field of this block. The first 32 bytes are reserved for vanity data that can be set using the [`--miner-extra-data`](../Reference/CLI/CLI-Syntax.md#miner-extra-data) command line option. Stores additional data when used with [Clique](../HowTo/Configure/Consensus-Protocols/Clique.md#genesis-file) and [IBFT](../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file).                                                                                                 |
```

Comment:
Is the `--miner-extra-data` option only used when using Ethash consensus protocol?

Reply:
No, it applies to all consensus protocols.